src/audio/base_fw.c

```diff
@@ -240,14 +240,16 @@ static uint32_t basefw_set_system_time(uint32_t param_id,
 				       uint32_t data_offset,
 				       const char *data)
 {
+	uint64_t current_dsp_time;
```

Comment:
This was not necessary, we don't need to be compatible with C89.

Reply:
thanks, updated